app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -9,4 +9,9 @@
   }
 
   @include breadcrumbs;
+
+  a[href^="#"] {
```

Comment:
I'd also prefer if this were a class since this is vague to how it's used. But pragmatically let's leave it as is for now.